airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/DebeziumConverterUtils.java

```diff
@@ -42,7 +43,10 @@ public static String convertDate(final Object input) {
     } else if (input instanceof Number) {
       return DataTypeUtils.toISO8601String(
           new Timestamp(((Number) input).longValue()).toLocalDateTime());
-    } else if (input instanceof String) {
+    } else if (input instanceof Date) {
+      return DataTypeUtils.toISO8601String(input)
```

Comment:
The `input` has `Object` type and can't be put as input parameter for the `DataTypeUtils.toISO8601String` without cast.